### DISCUSSION TITLE
Is there any keyboard shortcut to expand/collapse tabs?

### DISCUSSION THREAD

**Discussion by victor-marino** [zen-browser/desktop#995] [Q&A]
If I choose to expand tabs by default, I can see a button on the bottom left that allows me to collapse/expand them.

However, browsing through the keyboard shortcuts in the settings, I cannot find any that will trigger that action.

Is there any way to expand/collapse tabs through a keyboard shortcut?

**Comment by mnymkr** [comment]
I second this. A keyboard shortcut would help immensely.

**Comment by flandersen** [comment]
CTRL+ALT+C enables and disables compact mode.
